src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Controllers/Api/LegacyReleasesControllerTests.cs

```diff
@@ -0,0 +1,161 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils.MockUtils;
+using static Moq.MockBehavior;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Controllers.Api
+{
```

Comment:
Can you give an example of what you mean by this?

Reply:
Yep sure, there's an explanation of file-scoped namespaces here https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces.

In our codebase in most of the older code you will see a namespace declaration that looks a bit like this:

```csharp
// usings
using System;
...

namespace GovUk.Education.ExploreEducationStatistics.Project
{
    // types
    public class MyClass
    {
    }
}
```

But this has unnecessary indentation. Since c#10 you can now write this as:

```csharp
// usings
using System;
...

namespace GovUk.Education.ExploreEducationStatistics.Project;

public class MyClass
{
}
```